spec/compare_locales_spec.rb

```diff
@@ -0,0 +1,60 @@
+RSpec.describe CompareLocalesSupport do
+  let(:main_hash) {
+    {
+      'en'=> {
+        'app_name'=>'if me',
+        'app_description'=>'is a community for mental health experiences',
+        'email'=>'join.ifme@gmail.com',
+        'ellipsis'=>'[...]',
+        'created'=>'<strong>Created:</strong> %{created_at}',
+        'edited'=>'<strong>Created:</strong> %{created_at} <em>(edited)</em>',
+        'edited_updated_at'=>'<strong>Created:</strong> %{created_at} <em>(edited %{updated_at})</em>',
+        'salutation'=>'Hi %{name},',
+        'click_here'=>'click here',
+        'less'=>' [Less]',
+        'language'=>'Language'
+      }
+    }
+  }
+let(:second_hash) {
+  {
+    'en'=> {
+      'app_name'=>'if me',
+      'app_description'=>'is a community for mental health experiences',
+      'ellipsis'=>'[...]',
+      'created'=>'<strong>Created:</strong> %{created_at}',
+      'edited'=>'<strong>Created:</strong> %{created_at} <em>(edited)</em>',
+      'edited_updated_at'=>'<strong>Created:</strong> %{created_at} <em>(edited %{updated_at})</em>',
+      'salutation'=>'Hi %{name},',
+      'click_here'=>'click here',
+      'language'=>'Language'
+    }
+  }
+}
+  specify 'spanish locale file has nothing missing' do
+    english = './config/locales/en.yml'
+    spanish = './config/locales/es.yml'
```

Comment:
Probably better that you use the constants you defined on CompareLocalesSupport and DRY.
Since we will be changing how this works in a future version, for now just use `CompareLocalesSupport:: LOCALE_1`